static/app/views/issueList/overviewFc.spec.tsx

```diff
@@ -1068,15 +1070,34 @@ describe('IssueList', function () {
         },
       });
 
-      render(<IssueListOverview {...routerProps} {...props} />, {router});
+      const {rerender} = render(<IssueListOverview {...routerProps} {...props} />, {
+        router,
+      });
 
       await screen.findByRole('grid', {name: 'Create a search query'});
       await userEvent.click(getSearchInput());
       await userEvent.keyboard('foo{enter}');
 
-      expect(
-        await screen.findByText(/We couldn't find any issues that matched your filters/i)
```

Comment:
This test was actually just wrong before, these changes made it start failing because it should show "No issues match your search" instead